docs/src/Select/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import CodeBlock from "../components/CodeBlock";
+import ComponentExample from "../components/ComponentExample";
+import ComponentExampleWrapper from "../components/ComponentExampleWrapper";
+import ComponentWrapper from "../components/ComponentWrapper";
+import Select from "../../../src/Select/Select.js";
+import SelectOption from "../../../src/Select/SelectOption.js";
+import PropertiesAPIBlock from "../components/PropertiesAPIBlock";
+
+class SelectExample extends React.Component {
+  render() {
+    return (
+      <ComponentWrapper
+        title="Select"
+        srcURI="https://github.com/mesosphere/reactjs-components/blob/master/src/Select/Select.js"
+      >
+        <p className="lead flush-bottom">
+          Create a select component
+        </p>
+        <ComponentExampleWrapper>
+          <ComponentExample>
+            <div className="row row-flex">
+              <div className="column-6">
+                <Select
+                  name={"Select example"}
+                  className="dropdown-select"
+                  onChange={selectedItem => console.log("prop", selectedItem)}
+                >
+                  <SelectOption
+                    value="default"
+                    label="Select…"
+                    disabled={false}
+                    selected={true}
+                  />
+                  <SelectOption value="HOST" label="HOST">
+                    <strong>HOST</strong>
+                    <span>Recommended</span>
```

Comment:
I think we want to use the [Badge](http://mesosphere.github.io/cnvs/components/badges/) component here!